relay-metrics/src/lib.rs

```diff
@@ -7,14 +7,19 @@
 //!
 //! # Protocol
 //!
-//! Clients submit metrics in a [text-based protocol](Metric) based on StatsD. A sample submission
-//! looks like this:
+//! Clients submit metrics in envelopes containing a [text-based protocol](Metric) based on StatsD.
+//! A sample submission looks like this:
```

Comment:
Can we split this up into the text-based protocol, and then the Envelope protocol, like we had in Notion? It's good to keep the examples to the text portion of it, since that is StatsD compatible and we might want to introduce a new endpoint that takes this as plain text input.

Reply:
Done.